test/FunctionalTests/Tests/DataServices/UnitTests/ClientCSharpUnitTests/DataWebClientCSharp/LinqTests.cs

```diff
@@ -8035,24 +8035,73 @@ orderby t.TeamID
             }
             catch (Exception e)
             {
-                if (e.Message != "Can't add query option '$custom' because it begins with reserved character '$'.")
+                if (e.Message != "The query option '$custom' is not supported.")
                 {
                     throw new Exception("Test Failed");
                 }
             }
 
+            var queryableApplyOption = ((DataServiceQuery<League>)from l in context.CreateQuery<League>("Leagues")
+                                                        select l)
+                                                        .AddQueryOption("$apply", 1);
+
+            Trace.WriteLine("known, but unsupported query  options");
+
+            try

```

Comment:
It should be possible to convert these `try/catch`es to be `Assert.Throws` right?

Reply:
I don't think the assertion library used in these particular tests provides that method.